src/nsrt_learning/sampler_learning.py

```diff
@@ -274,9 +296,14 @@ def _create_sampler_data(
                     continue
                 # Add this datapoint to the negative data.
                 negative_data.append((state, sub, option, goal))
-    logging.info(f"Generated {len(positive_data)} positive and "
-                 f"{len(negative_data)} negative examples")
-    assert len(positive_data) == len(datastores[datastore_idx])
+
+                if len(negative_data
+                       ) >= CFG.sampler_learning_max_negative_data:
```

Comment:
nit: change >= to > here so that yapf doesn't make formatting weird

Reply:
can we keep the `>=`? my test kind depends on it...